app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/CheckAgencyController.scala

```diff
@@ -59,13 +59,8 @@ class CheckAgencyController @Inject()
   }
 
   val showCheckAgencyStatus: Action[AnyContent] = AuthorisedWithSubscribingAgent {
```

Comment:
We need everything that is provided in AuthorisedWithSubscribingAgent. If we create another builder it will need everything in the existing method except for the one line below, do you think it is necessary producing duplicate code that excludes the enrolment check shown in the code below? 
  ` case true => enrolments map { e => body(authContext)(AgentRequest(e, request)) }`
`